deploy/handler/operator.yaml

```diff
@@ -23,6 +23,9 @@ spec:
       nodeSelector:
         beta.kubernetes.io/arch: amd64
         node-role.kubernetes.io/master: ""
+      {{range $key, $value := .HandlerNodeSelector}}
+        {{$value | printf "%s: %s" $key }}
```

Comment:
What about `{{ $key -}}: {{- $value }}` although dealing with new lines is a pain...